prometheus/vars/main.yaml

```diff
@@ -22,6 +22,11 @@ __federated_scrape_config:
   static_configs:
   - targets: '{{ (prometheus__federated_prometheus or []) | list }}'
 
+__alertmanager_scrape_config:
```

Comment:
This is good, but it won't do anything unless you add this variable to `__all_scrape_configs`